components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -394,6 +394,14 @@ Rules of type `Node` can have conditions with attributes `Chef Organization`, `C
 Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
 filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.
 
+#### Deleting Projects
+
+To delete a project, navigate to the Projects list under the **Settings** tab. Find the project you wish to delete in the list, click the menu on the
```

Comment:
Do we normally capitalize resources?

Reply:
mmm, reading the paragraph above, looks like we refer to the "project list page" instead of "Projects list"

Reply:
Looks like no on resource capitalization.